app/Repositories/User/UserRepository.php

```diff
@@ -24,12 +24,22 @@ public function getModel()
      */
     public function uploadAvatar($request, $name, $oldAvatarName, $pathUpload)
     {
-        $pathFile = $oldAvatarName;
-
         if (Storage::disk('local')->exists($oldAvatarName) && $oldAvatarName != config('settings.avatar_default_path')) {
             Storage::disk('local')->delete($oldAvatarName);
         }
 
         return $this->uploadUserAvatar($request, $name, $pathUpload);
     }
+
+    /**
+     * @param $keyword
+     * @return UserRepository
+     */
+    public function likeSearch($keyword)
+    {
+        $this->model = $this->model->where('name', 'LIKE', '%' . $keyword . '%')
```

Comment:
return $this->model->where('name', 'LIKE', '%' . $keyword . '%')->orWhere('email', 'LIKE', '%' . $keyword . '%');
được ko a

Reply:
:D đc chứ